### ISSUE TITLE
pretrained backbone

### ISSUE THREAD

**Issue by mirecta** [zhen8838/K210_Yolo_framework#9]
Hi can i use pretrained backbone mobilenet darknet network as feature extractor,  freeze this layers and train only detector layers ? AKA transfer learning ?

**Comment by zhen8838** [comment]
yes , you can. but mobilenet backbone for k210 need to modify. you can refer my code `https://github.com/zhen8838/K210_Yolo_framework/blob/master/models/keras_mobilenet.py`
